felix/fv/ipip_test.go

```diff
@@ -404,6 +405,17 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ IPIP topology before adding
 				ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 				defer cancel()
 				c, err := client.FelixConfigurations().Get(ctx, "default", options.GetOptions{})
+				if err != nil {
+					// Create the default config if it doesn't already exist.
+					if _, ok := err.(cerrors.ErrorResourceDoesNotExist); ok {
+						c = api.NewFelixConfiguration()
+						c.Name = "default"
+						c, err = client.FelixConfigurations().Create(ctx, c, options.SetOptions{})
+						Expect(err).NotTo(HaveOccurred())
+					} else {
+						Expect(err).NotTo(HaveOccurred())
```

Comment:
This is a little confusing, because we've already asserted that err != nil in order to reach this branch.

Reply:
We also assert that the error didn't occur a second time on line 419 just below

Reply:
I apologize for this mess :see_no_evil: The else statement does nothing, will clean it up. Thanks!

Reply:
Still not 100% satisfied but I think it's clearer now. Please let me know if this is still not good. Thanks!